goal_src/engine/camera/cam-states.gc

```diff
@@ -1213,14 +1213,20 @@
 (deftype cam-string-bank (basic)
   ((los-coll-rad  meters  :offset-assert   4)
    (los-coll-rad2 meters  :offset-assert   8)
+   (y-adjust-ratio float)
+   (z-adjust-ratio float)
    )
   :method-count-assert 9
-  :size-assert         #xc
-  :flag-assert         #x90000000c
+ ; :size-assert         #xc
+ ; :flag-assert         #x90000000c
   )
 
 
-(define *CAM_STRING-bank* (new 'static 'cam-string-bank :los-coll-rad (meters 1) :los-coll-rad2 (meters 0.5)))
+(define *CAM_STRING-bank* (new 'static 'cam-string-bank
+                               :los-coll-rad (meters 1)
+                               :los-coll-rad2 (meters 0.5)
+                               :y-adjust-ratio 1.0
+                               :z-adjust-ratio 1.0))
```

Comment:
I don't think you meant to push this?

Reply:
yeah my bad was staging pieces of the file before